### PR TITLE
Remove most of README.md as it refers to the S3 broker

### PR DESCRIPTION
**Pull request by risicle**
This is really confusing otherwise.

Keep a very basic description as a lot of the details aren't finalized yet (?)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,150 +1,5 @@
...
-```make
-make test
-```
-
-The integration tests will require you to have at least the IAM permissions listed in the above [requirements](#requirements) section.
-
-## `costs_by_month` utility
-
-In `cmd/costs_by_month/README.md` you can find instructions for calculating the cost of tenant S3 buckets over the last few months.
+The implementation creates an S3 bucket for every service instance and bindings are implemented as an IAM user with access keys.
```
Comment (Krenair) [inline README.md:5]: S3 bucket?

Comment (risicle) [reply README.md:5]: Oh how did that get left in

**Review by Krenair — DISMISSED** [review]

**Comment by risicle** [comment]
Ah need to set up my signed commits.

**Comment by risicle** [comment]
omgggg that was annoying.

Gonna need a re-approval now ;)

**Review by chrisfarms — APPROVED** [review]

**Pull request merged**